pkg/dht/dht.go

```diff
@@ -7,14 +7,15 @@ import (
 	"context"
 	"time"
 
+	"storj.io/storj/pkg/overlay"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/storj"
 	"storj.io/storj/storage"
 )
 
 // DHT is the interface for the DHT in the Storj network
 type DHT interface {
-	FindNear(ctx context.Context, start storj.NodeID, limit int, restrictions ...pb.Restriction) ([]*pb.Node, error)
```

Comment:
How are we filtering for restrictions now?

Reply:
We don't filter restrictions/capacity in DHT as this info is not available in DHT anymore. We now filter by capacity when selecting nodes from overlay cache.